MAINTAINERS.md

```diff
@@ -0,0 +1,165 @@
+rather than having a mentorship process where a new maintainer is guided by a
+senior member, we want to see the capabilities of the candidate before being
+accepted.
+
+**TODO:** Detail the meaning of a senior project maintainer with a requirement
+along the lines of more than 6 months experience as a project maintainer. Also
+related to contributor ladder.
+
+## Maintainership Application
+
+Once you have satisfied the listed maintainership requirements, you can submit
+your application to kubernetes-wg-contribex@googlegroups.com wherein the
+application will be discussed by the contributor leads (top-level **OWNERS**) and
+put to vote. If the vote result is negative, do not be discouraged! You will be
+given detailed feedback so that you can improve yourself and apply again. We
+want to empower our community and cultivate a trust relationship that's why we
+carefully evaluate abilities and experience of each candidate.
+
+### Application Template
+
+A rudimentary application template is as follows:
+
+```yaml
+github-id: foo-bar
+
+# List of senior maintainers that agreed to sponsor you.
+sponsors:
+  - <github-id>
+  - ...
+  - ...
+  
+# Links to enumerated list of your contributions for ease of review
+# Note: It'll get easier to create such lists as we improve our contribution measurement tooling
+# Note: The following template is just and example, you can use any formatting, however the important
+#       thing is to list your contributions and classify them by repository/type/theme to present a
+#       digestable view of your efforts to the reviewers of your application.
+contributions:
+  - X merged commits <github-link>
+    - X/3 in sig-foo
+    - 2X/3 in overall documentation
+  - Y reviewed PRs <github-link>
+    - Y in kubectl
+
+# List of previous and current responsibilities in the Kubernetes organization.
+responsibilities:
+  current:
+    - sig-X contributor
+  previous:
+    - v1.4 release czar
+    
+# List of additional repositories that you need write access to if there is any
+additional-write-access-to:
+  - kubernetes/client-go
+  - kubernetes/contrib
+  
+# Optional access duration, omitable if there is no planned end date.
+duration: the v1.6 release | for 3 months | etc.
+```
+
+## What we expect from our maintainers
+
+First and foremost, be a good person, as cheeky as it sounds, you are now a
+member of the Kubernetes project and thus represent the project and your fellow
+developers. We value our community tremendously and we'd like to keep
+cultivating a friendly and collaborative environment for our
+members/contributors/users. For more,
+see
+[CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
+Good technical judgement, and knowledge over one or more components of
+Kubernetes, it might be a stand-alone project under the Kubernetes GitHub
+organization (e.g. [minikube](github.com/kubernetes/minikube)) or it might be a
```

Comment:
Does this mean it's possible to become a maintainer without write access to `kubernetes/kubernetes`, or is this repo a requirement for maintainers that work on other stand-alone projects?